website/client/components/inventory/drawer.vue

```diff
@@ -75,6 +77,24 @@
   width: 100%;
   overflow-x: scroll;
   white-space: nowrap;
+  position: relative;
+
+  & .message {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    top: calc(50% - 30px);
```

Comment:
it should http://caniuse.com/#feat=calc , back then the browsers need to have their own prefix but now all supports it